Dockerfile

```diff
@@ -67,7 +67,7 @@ ENV PACKAGES=git-http
+ENV PACKAGES="${PACKAGES} avahi-daemon avahi-daemon-service-ssh avahi-daemon-service-http base-files bash blkid block-mount busybox curl cryptsetup dnsmasq dropbear ca-bundle ca-certificates e2fsprogs f2fs-tools firewall fstools gcc git ip6tables iptables iw i2c-tools jshn jsonfilter kernel kmod-ath kmod-ath9k-common kmod-cfg80211 kmod-crypto-xts kmod-crypto-iv kmod-crypto-misc kmod-crypto-user kmod-fs-ext4 kmod-fs-f2fs kmod-gpio-button-hotplug kmod-i2c-algo-bit kmod-i2c-bcm2708 kmod-i2c-core kmod-ip6tables kmod-ipt-conntrack kmod-ipt-core kmod-ipt-nat kmod-ledtrig-netdev kmod-lib-crc-ccitt kmod-mac80211 kmod-nf-conntrack kmod-nf-conntrack6 kmod-nf-ipt kmod-nf-ipt6 kmod-nf-nat kmod-nf-nathelper kmod-nls-base kmod-ppp kmod-pppoe kmod-pppox kmod-slhc kmod-usb-acm kmod-usb-core kmod-usb2 kmod-usb-dwc2 kmod-usb-gadget-eth kmod-usb-net kmod-usb-net-cdc-ether kmod-usb-net-rndis less libblobmsg-json libgmp libip4tc libip6tc libiwinfo libiwinfo-lua libjson-c libjson-script liblua libnl-tiny libubox libubus libubus-lua libuci libuci-lua libustream-openssl libxtables libyaml lua luci luci-app-firewall luci-app-splash luci-base luci-lib-ip luci-lib-nixio luci-mod-admin-full luci-proto-ipv6 luci-proto-ppp luci-theme-material luci-theme-openwrt make mtd netifd obfsproxy opkg ppp ppp-mod-pppoe procd python-crypto python-dev python-pip python-pyptlib python-smbus python-yaml resize2fs rpcd sfdisk swconfig tcpdump tor tor-geoip twisted uboot-envtools ubox ubus ubusd uci uhttpd uhttpd-mod-ubus usign vim wpad-mini zope-interface nodogsplash openssh-sftp-server"
```

Comment:
I get the `nodogsplash`, but do we need the `openssh-sftp-server` for anything, or is that just a debugging addition?

Reply:
Yeah that was just to be able to shh into it.

Reply:
Any more improvements to this?

Reply:
I've added a page to fetch and apply the tor config from the REST api. The next steps are to allow the config to be fetched from an onion address and then to setup the `dfpk-wifi` page as a forward authenticating service for `nodogsplash`.

Reply:
I see some changes to use tor as a socks5 proxy... is this ready for me to try yet?

Reply:
Yes, socks5 proxy is good to go.